monitoring/grafana/jsonnet/panels/compactor.libsonnet

```diff
@@ -0,0 +1,163 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+local row = grafana.row;
+local timeseriesPanel = grafana.timeseriesPanel;
+local prometheus = grafana.prometheus;
+
+local span = import 'mixon/span.libsonnet';
+local strings = import 'lib/strings.libsonnet';
+
+local panels = {
+  compaction_success_failure_counts_overview::
+    local expr = |||
+      sum(storage_level_compact_frequency{namespace="$namespace", risingwave_name="$instance"}) by (result)
+    |||;
+
+    timeseriesPanel.new(
+      title='Compaction Success & Failure Counts (Overview)',
+    ).addTarget(
+      prometheus.target(
+        expr=strings.trim(expr),
+        legendFormat='{{ result }}',
+      )
+    ) + span.span('half'),
+
+  compaction_success_failure_counts_detailed::
+    local expr = |||
+      sum(storage_level_compact_frequency{namespace="$namespace", risingwave_name="$instance"}) by (pod, group, result)
+    |||;
+
+    timeseriesPanel.new(
+      title='Compaction Success & Failure Counts (Detailed)',
+    ).addTarget(
+      prometheus.target(
+        expr=strings.trim(expr),
+        legendFormat='{{ pod }} - group{{ group }} - {{ result }}',
+      )
+    ) + span.span('half'),
+
+<<<<<<< HEAD
+    max_compaction_task_duration_overview::
+=======
+  max_compaction_task_duration::
+>>>>>>> 86d6f4b (style: format the jsonnet files)
```

Comment:
Find a conflict not resolved here 🤣

Reply:
oh interesting. it was able to compile so I thought there was no issues 🤣